synth.py

```diff
@@ -146,7 +146,6 @@
 
 os.rename("dev/.gitignore", ".gitignore")
 os.rename("dev/.eslintignore", ".eslintignore")
-os.rename("dev/.eslintrc.json", ".eslintrc.json")
```

Comment:
why are we removing this line again?

Reply:
We probably want to keep our override that disallows describe.only.